src/nvim/buffer.h

```diff
@@ -4,6 +4,7 @@
 #include "nvim/window.h"
 #include "nvim/pos.h"  // for linenr_T
 #include "nvim/ex_cmds_defs.h"  // for exarg_T
+#include "nvim/screen.h"  // for StlClickRecord
```

Comment:
This is not needed by buffer.h directly, shouldn't the various .c file(s) add this include explicitly? ("include-what-you-use")